packages/ui-extensions/src/surfaces/admin/components/patterns/examples/setupGuide.html

```diff
@@ -0,0 +1,161 @@
+<s-section id="setup-guide-section">
+  <s-grid gap="base">
+    <!-- Header -->
+    <s-grid gap="small-200">
+      <s-grid gridTemplateColumns="1fr auto auto" gap="small-300" alignItems="center">
+        <s-heading>Setup Guide</s-heading>
+        <s-button
+          accessibilityLabel="Dismiss Guide"
+          onClick="window.puzzleApp.dismissGuide(document.getElementById('setup-guide-section'))"
+          variant="tertiary"
+          tone="neutral"
+          icon="x"
+        ></s-button>
+        <s-button
+          id="toggle-guide-button"
+          accessibilityLabel="Toggle setup guide"
+          onClick="window.puzzleApp.toggleGuide(this, document.getElementById('steps-container'))"
+          variant="tertiary"
+          tone="neutral"
+          icon="chevron-up"
+        ></s-button>
+      </s-grid>
+      <s-paragraph>
+        Use this personalized guide to get your store ready for sales.
+      </s-paragraph>
+      <s-stack direction="inline" gap="small-200" alignItems="center">
+        <s-paragraph id="progress-text" tone="subdued">0 out of 3 steps completed</s-paragraph>
```

Comment:
Is this a real `tone` for paragraph?

Reply:
I have a general comment about all of them. We seem to be setting `id` on random things. For things that need ids it makes sense (for example, when we use tooltips), but besides that I kind of think we shouldn't set any ids in our examples. If people copy/paste, we may be setting them up for failure since you can only have 1 element with an id on the page.